HousingSearchApi/V1/Boundary/Requests/GetTenureListRequest.cs

```diff
@@ -4,7 +4,6 @@ namespace HousingSearchApi.V1.Boundary.Requests
 {
     public class GetTenureListRequest : HousingSearchRequest
     {
```

Comment:
Do we still need this class?

Reply:
It was there as an empty class before so I left it.